GNUmakefile

```diff
@@ -8,6 +8,7 @@ install: build
 
 # See https://golangci-lint.run/
 lint:
+	golangci-lint cache clean
```

Comment:
Interesting, I've never needed to run this before. Is there something going on that requires it?

Reply:
I haven't experienced this before, but I have had a situation with the `unparam` linter where it wouldn't pick up an issue until I cleaned the cache.

I might be something that can only happen locally, as the runners on the GH CI are always executed against a fresh environment.

I'm not sure it's necessary, but it did seem to make a difference in spotting [this](https://github.com/hashicorp/terraform-provider-tls/pull/209/files#diff-cba759d368325cfd69f2c0b0bc003b2d4bee940fd86255e00549060ef711e175R48). 🤷